cpp/test/Slice/errorDetection/CaseInsensitive.err

```diff
@@ -24,10 +24,8 @@ CaseInsensitive.ice:146: dictionary 'D' differs only in capitalization from dict
 CaseInsensitive.ice:147: redefinition of module 'm1' as dictionary
 CaseInsensitive.ice:148: dictionary 'M1' differs only in capitalization from module 'm1'
 CaseInsensitive.ice:151: enumeration 'eN1' differs only in capitalization from enumeration 'en1'
-CaseInsensitive.ice:151: 'eN1' has changed meaning
```

Comment:
The real error was that `eN1` and `en1` conflict with one another.
Not that `eN1` changed meaning...